VERSIONING.md

```diff
@@ -33,27 +172,42 @@ File types in question:
   - Ontology files
     - TTL
     - HTML
-  - TypeScript types
-  - UML Diagrams
+  - TypeScript types: Needs input from Scripting API TF what major, minor, patch means for type versions.
+  - UML Diagrams: How a UML diagram is versioned needs further input. No input from Ege, Daniel or Luca so far
 
 Notes: 
 - @mahdanoura : Also, the deprecation of terms needs to be considered, i.e. not removing but marking as deprecated.
 - Best practices for RDF versioning should be studied. See https://www.w3.org/TR/prov-primer/
 
-#### Reflecting Meaning on Version Information
+### Reflecting Meaning of Version Information
 
 How do we reflect a patch change on the artifacts? Do we use semantic versioning, use date, and ignore semantic versioning, do we use alpha, beta etc.
 
 - ( @lu-zero ) A date afterward would be good "enough" for implementers. A release every month (in addition to semver) would also be good. We can also pipeline it and tag each resource exposed to github.io. 
 - ( @mjkoster ) Short commit hash would be fine. Monthly release may not make sense (unless there is a need)
 - ( @mmccool ) To make things easy we should map our directories to the url structure. We should make all URLs are versioned
 - @mahdanoura : According to OWL2 specification, for every ontology we can define an Ontology IRI (generic reference, should stay the same through versions) and a Ontology Version IRI. The difference is during dereferencing of the IRIs, the ontology IRI should redirect to the most recent version and the version IRI to the specific version.
-
-#### Synchronisation of Changes
+- @egekorkan: we can do scoped versioning : td-version:resource-version -> 2.0.0.pre1+1.0.0.pre1 but this can be complicated. -> Not in favor of it anymore after talking
+  - others: Something like a changelog in the discovery can convey the meaning of change. Using one version would be simpler.
+- @egekorkan: We cannot do semantic versioning in the usual sense since TD 2.0 has set the expectation already. Or we decouple the spec version from resource version, e.g. TD 2.0 can have resources in version 25.2.1
+- Cris: CSS simply increases the major version -> TD 2 has TD 2 resources. Until we get there, we can have TD 2.pre1 TD 2.a1.
+- Luca: We can embed resource/format specific version inside the resource itself.
+- Luca: We assume the TD 2.0 will not change but the resources can have big changes since they are moving targets. E.g. JSON Schema Draft 7 becomes obsolete and we decide to update to JSON Schema 2026-09 with totally new terms etc. In this case, JSON Schema resource gets a major version bump.
+  - In this case, the resource needs its own semantic versioning lifecycle but refer to which spec version it refers to. Example for JSON Schema version: 4.0.0+2.0 becomes 5.0.0+2.0 following the example above. Next TD version, we will get 4.0.0+2.0 -> 5.0.0+3.0 
```

Comment:
```suggestion
- @egekorkan: we can do scoped versioning: `td-version:resource-version` -> `2.0.0.pre1+1.0.0.pre1`, but this can be complicated. -> Not in favor of it anymore after talking
  - others: Something like a changelog in the discovery can convey the meaning of change. Using one version would be simpler.
- @egekorkan: We cannot do semantic versioning in the usual sense, since TD 2.0 has already set the expectation. Or we decouple the spec version from the resource version, e.g., TD 2.0 can have resources in version `25.2.1`.
- Cris: CSS simply increases the major version -> TD 2 has TD 2 resources. Until we get there, we can have TD 2.pre1 TD 2.a1.
- Luca: We can embed resource/format specific version inside the resource itself.
- Luca: We assume the TD 2.0 will not change, but the resources can have big changes since they are moving targets, e.g., `JSON Schema Draft 7` becomes obsolete and we decide to update to `JSON Schema 2026-09` with totally new terms, etc. In this case, JSON Schema resource gets a major version bump.
  - In this case, the resource needs its own semantic versioning lifecycle but refer to which spec version it refers to. Example for JSON Schema version: `4.0.0+2.0` becomes `5.0.0+2.0` following the example above. Next TD version, we will get `4.0.0+2.0` -> `5.0.0+3.0`.
```